src/components/UI/select/Select.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { FormControl, MenuItem, Select as SelectM } from "@mui/material";
+import { ISelectProps } from "components/UI/select/typesDef";
+import { Controller } from "react-hook-form";
+import { SErrorMessage } from "components/UI/textField/style";
+
+function Select(props: ISelectProps) {
+  const { options, control, formFieldName, errors } = props;
+  const sxProps = {
```

Comment:
why we use it as variable here? why its not in styles?

Reply:
fixed